internal/usage/usage.go

```diff
@@ -143,10 +143,10 @@ dbName and collection are required only for built-in roles.`
 	NotificationToken                         = "Slack API token, Bot token, or Flowdock personal API token." //nolint:gosec // This is just a message not a password
 	NotificationsChannelName                  = "Slack channel name. Required for the SLACK notifications type."
 	AlertConfigAPIKey                         = "Datadog API Key, Opsgenie API Key, or VictorOps API key. Required if the notificationType is DATADOG, OPS_GENIE, or VICTOR_OPS, respectively." //nolint:gosec // This is just a message not a credential
-	APIKey                                    = "API Key."
-	DatadogAPIKey                             = "Datadog API key that allows Atlas to access your Datadog account."               //nolint:gosec // This is just a message not a credential
-	OpsgenieAPIKey                            = "Opsgenie API key that allows Atlas to access your Opsgenie account."             //nolint:gosec // This is just a message not a credential
-	VictorOpsAPIKey                           = "Splunk On-Call API key that allows Atlas to access your Splunk On-Call account." //nolint:gosec // This is just a message not a credential
+	APIKey                                    = "Unique 24-digit string that identifies your API key."                                                                                          //nolint:gosec // This is just a message not a credential
```

Comment:
```suggestion
	APIKey                                    = "Unique 24-hexadecimal digit string that identifies your API key."                                                                                          //nolint:gosec // This is just a message not a credential
```

Reply:
[q] Why did you add this usage? is this for public or private keys? Where is it used?

Reply:
Hi @andreaangiolillo, these 4 items already existed in the usage file, I just changed the descriptions for all of them and the make fmt added some space between the description and the nolint note, so they're showing as removed and re-added. This particular example just replaces `API Key.` with `Unique 24-digit string that identifies your API key.` for the existing item.